plugins/ucrm-esignature-platform/src/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "1",
+    "information": {
+        "name": "contract-plugin",
+        "displayName": "eSignature Platform - 07INTERNET.RO",
+        "description": "You can visualize and sign your contract directly on the platform.",
+        "url": "https://github.com/matei1337/ucrm-esignature-platform",
```

Comment:
it might be better to link to `https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/ucrm-esignature-platform` in the official repository as that will make sure people will get information for the version they're probably running